runtime/plugin/man.vim

```diff
@@ -1,6 +1,21 @@
-if get(g:, 'loaded_man', 0)
+if exists('g:loaded_man')
   finish
 endif
 let g:loaded_man = 1
 
-command! -count=0 -nargs=+ Man call man#get_page(<count>, <f-args>)
+if !exists("g:find_man_window")
+  let g:find_man_window = 1
+endif
+
+if !exists('g:man_synopsis')
+  let g:man_synopsis = 'SYNOPSIS'
+endif
+
+command! -bang -complete=customlist,man#Complete -nargs=* Man call
+      \ man#get_page(<bang>0, 'edit', <f-args>)
+command! -bang -complete=customlist,man#Complete -nargs=* Sman call
+      \ man#get_page(<bang>0, 'split', <f-args>)
+command! -bang -complete=customlist,man#Complete -nargs=* Vman call
+      \ man#get_page(<bang>0, 'vsplit', <f-args>)
+command! -bang -complete=customlist,man#Complete -nargs=* Tman call
+      \ man#get_page(<bang>0, 'tabe', <f-args>)
```

Comment:
No, behavior should remain `:tabnew` by default. That is the best approximation we have for "pager" behavior which preserves window layout, for the time being.

I suggest a slight change, however:

    :execute (tabpagenr()-1).'tabnew'

This opens tab _before_ current tab, so that `q` returns to the original tab. Again, to approximate traditional behavior of `:!man`.

Also let's keep any new commands for a separate PR. I want to move quickly on this one ;)